### DISCUSSION TITLE
Limit the size of Elasticsearch volume?

### DISCUSSION THREAD

**Discussion by EgonHeuson** [nextcloud/all-in-one#2439] [Questions] [answered]
Hello the NextcloudAIO community! I hope you are doing very well!
I had an issue on one of the NCAIO instances I run due to the Fulltextsearch option. If I'm correct, it uses ElasticSearch, and the size of the volume created by the later has reached 20GB, which made the disk of the VM completely full. I temporarily solved the problem by increasing the size of the disk for the VM, but I wondered if it would be possible to limit the size of the volume created by ElasticSearch? Or maybe it is not possible if I want the function to work properly? In this case, any idea on the classical size for this volume? 
Thanks in advance!
Have a nice weekend!

**Comment by szaimen** [comment, marked as answer]
> I temporarily solved the problem by increasing the size of the disk for the VM, 

This is the corrext way of solving this problem.

> but I wondered if it would be possible to limit the size of the volume created by ElasticSearch? 

You cannot limit the size as otherwise FTS would not work properly anymore.

> Or maybe it is not possible if I want the function to work properly? 

Yes, correct.

> In this case, any idea on the classical size for this volume?

It really depends on the amount of files that you put into your Nextcloud so I cannot tell you what a classical size is. But for a lot of files, 20GB FTS volume does not sound unreasonable.

---


Actially there is a way to limit the fts volume size: disable FTS from the AIO interfacs.

**Reply by EgonHeuson** [reply]
Ok thanks! :-) Yes I have a lot of data on this Nextcloud (1To nearly). I wondered if it would be like 20Go, 50Go or even 100Go that would be required by FTS?

**Reply by szaimen** [reply]
I fear I cannot give you exact numbers. You will need to find out yourself.

**Reply by EgonHeuson** [reply]
ok :-) Thanks! ☺️
